src/widgets/allergies/allergy-intolerance.resource.tsx

```diff
@@ -31,13 +31,13 @@ export function fetchAllergyByUuid(allergyUuid: string) {
 function mapAllergyProperties(allergy) {
   let manifestations: Array<string> = [];
   allergy?.reaction[0]?.manifestation?.map(coding =>
-    manifestations.push(coding.coding[1].display)
+    manifestations.push(coding.coding[0].display)
```

Comment:
Sure `[0]?.display` isn't more suitable?

Reply:
It is indeed.